libs/perfecto_libs/iOS_lib.py

```diff
@@ -3472,6 +3472,27 @@ def getDeviceModelName(setup_perfectoMobile):
     print("ModelName: " + deviceModel)
     return deviceModel
 
+def speedtest_app(request, setup_perfectoMobile,get_APToMobileDevice_data):
+    report = setup_perfectoMobile[1]
+    driver = setup_perfectoMobile[0]
+    driver.switch_to.context('NATIVE_APP')
+    connData = get_APToMobileDevice_data
+    time.sleep(5)
+    report.step_start("Get upload/Download Speed from speed test app")
+    openApp( 'com.ookla.speedtest',driver)
```

Comment:
Can you change:
**openApp( 'com.ookla.speedtest',driver)**
to:
**openApp( 'com.ookla.speedtest',setup_perfectoMobile)**